.github/workflows/code-standards.yaml

```diff
@@ -133,3 +133,27 @@ jobs:
 
       - name: run unit tests
         run: vendor/bin/phpunit
+
+  end-to-end-tests:
+    name: end-to-end tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: start the site
+        run: docker compose up -d
+
+      # Give the containers a moment to settle.
+      - name: wait a tick
+        run: sleep 10
+
+      - name: populate the site
+        run: |
+          cp web/sites/example.settings.dev.php web/sites/settings.dev.php
+          make install-site
+          docker compose exec drupal drush content:import scs-export/node-page-427c45cf-9fb6-4e79-a0f5-5a54f82b0bfb.yml
```

Comment:
Will this yaml file name change?

Reply:
Yes. After #277 is merged, I'd want to come back and remove line 156 here, as `make install-site` will handle loading content in a more generic way.